### PR TITLE
Dropdowns, adding BatchRead operation and fixing issue #2 for analysis

### PR DESCRIPTION
**Pull request by nrobert**
Changing free text input to Dropdown when the values are limited by the API.
Adding Batch Read operation for OCR processing.
Fix for #2 by disabling Brands and Objects visual features if analysis language is not 'en'

### CODE REVIEW

File: src/Orneholm.CognitiveWorkbench.Web/Services/ImageComputerVisionAnalyzer.cs [hunk 1]
```diff
@@ -71,17 +72,28 @@ public async Task<ComputerVisionAnalyzeResponse> Analyze(string url, string anal
...
 
-        private async Task<ImageAnalysis> ComputerVisionAnalyzeImage(string url, string analysisLanguage)
+        private async Task<ImageAnalysis> ComputerVisionAnalyzeImage(string url, AnalysisLanguage analysisLanguage)
         {
+            switch (analysisLanguage)
+            {
+                case AnalysisLanguage.en:
+                    break;
+                default:
+                    AnalyzeVisualFeatureTypes.Remove(VisualFeatureTypes.Brands);
```
Comment (PeterOrneholm) [inline src/Orneholm.CognitiveWorkbench.Web/Services/ImageComputerVisionAnalyzer.cs]: This modifies the shared (static) variable. I'd suggest tat you split the visual features into two static fields. One that works in all languages, and for the english ones. The create a local list that picks the relevant values.

File: src/Orneholm.CognitiveWorkbench.Web/Views/Vision/ComputerVision.cshtml [hunk 2]
```diff
@@ -339,6 +351,62 @@
...
+        {
+            <section class="cwb-result-computervision-analyze">
+                <h2 class="mt-5">Computer Vision - Batch Read</h2>
+
+                <div class="row">
+                    <div class="col-md-7">
+                        <div class="card mb-3 cwb-result-image">
+                            <img src="@Model.ComputerVisionAnalyzeResponse.ImageInfo.Url" class="card-img-top" alt="@Model.ComputerVisionAnalyzeResponse.ImageInfo.Description" />
+                            <div class="cwb-image-boxes">
+                                <div class="cwb-image-boxes-ocr-lines">
```
Comment (PeterOrneholm) [inline src/Orneholm.CognitiveWorkbench.Web/Views/Vision/ComputerVision.cshtml]: Use something unique for this, like `cwb-image-boxes-batchread-lines`

**Review by PeterOrneholm — CHANGES_REQUESTED** [review]
Nice. One small thing trgarding css class. It's that class that connects the hover on a row with highlighting that square.

File: src/Orneholm.CognitiveWorkbench.Web/Views/Vision/ComputerVision.cshtml (hunk 2 above)
Comment (nrobert) [reply src/Orneholm.CognitiveWorkbench.Web/Views/Vision/ComputerVision.cshtml]: Shame on me, I missed this one! I thought I changed them after your comment on PR number 1

File: src/Orneholm.CognitiveWorkbench.Web/Services/ImageComputerVisionAnalyzer.cs (hunk 1 above)
Comment (nrobert) [reply src/Orneholm.CognitiveWorkbench.Web/Services/ImageComputerVisionAnalyzer.cs]: Good catch, updated

File: src/Orneholm.CognitiveWorkbench.Web/Views/Vision/ComputerVision.cshtml (hunk 2 above)
Comment (nrobert) [reply src/Orneholm.CognitiveWorkbench.Web/Views/Vision/ComputerVision.cshtml]: And I also rollbacked by error the fix on recognizeText class. Everything should be ok right now

**Comment by PeterOrneholm** [comment]
Looks good, thanks!

**Pull request merged**